cypress/e2e/registration-listspec.cy.js

```diff
@@ -0,0 +1,32 @@
+describe('empty spec', () => {
+  it('passes', () => {
+    cy.visit('http://localhost:3000/registration-list')
+  })
+})
+
+describe('registration list page', () => {
+  it('displays the title', () => {
+    cy
+    .visit('http://localhost:3000/registration-list')
+    .contains('A list of things you will need to complete the registration')
+  })
+  it('displays first element of unordered list', () => {
+    cy
+    .visit('http://localhost:3000/registration-list')
+    .get('ul li:first')
+  })
+  it('displays last element of unordered list', () => {
+    cy
+    .visit('http://localhost:3000/registration-list')
+    .get('ul li:last')
+  })
+})
+
+describe('continue button', () => {
```

Comment:
nit: Although `describe` and `context` blocks are essentially the same apparently, maybe this could go inside a `context` block within the `describe`. (Like in this example: https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Test-Structure)